eth-types/src/evm_types.rs

```diff
@@ -200,7 +200,7 @@ impl GasCost {
     /// Per-pair gas cost for BN256 pairing
     pub const PRECOMPILE_BN256PAIRING_PER_PAIR: Self = Self(34_000);
     /// Base gas cost for precompile call: MODEXP
-    pub const PRECOMPILE_MODEXP: Self = Self(0);
+    pub const PRECOMPILE_MODEXP: Self = Self(200); // eip2565
```

Comment:
i think this change is correct.  Then why the old codes worked well..? @roynalnaruto

Reply:
I think we were using `PRECOMPILE_MODEXP_MIN` which is defined below